src/main/java/com/example/tournamentMaker/excel/football/FootballLeagueWriterService.java

```diff
@@ -0,0 +1,187 @@
+package com.example.tournamentmaker.excel.football;
+
+import com.example.tournamentmaker.excel.ExcelStrategy;
+import com.example.tournamentmaker.excel.ExcelUtil;
+import com.example.tournamentmaker.statistics.StatisticService;
+import com.example.tournamentmaker.statistics.Statistics;
+import com.example.tournamentmaker.tournament.Tournament;
+import lombok.RequiredArgsConstructor;
+import org.apache.poi.ss.usermodel.*;
+import org.apache.poi.xssf.usermodel.XSSFFont;
+import org.apache.poi.xssf.usermodel.XSSFRichTextString;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.springframework.stereotype.Service;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class FootballLeagueWriterService implements ExcelStrategy {
+    private String filePath = "D:\\tournamentMaker.xlsx";
+    private static final String SHEET_NAME = "League";
+    private final FootballStatisticWriterService footballStatisticWriterService;
+    private final Workbook workbook = new XSSFWorkbook();
+    private final Sheet sheet = workbook.createSheet(SHEET_NAME);
+    private static final int TEAM_COLUMN_EXCEL_WIDTH = 26;
+    private static final int COLUMN_WITH_NUMBER_EXCEL_WIDTH = 5;
+    private static final int LAST_5_COLUMN_EXCEL_WIDTH = 12;
+    private static final int STATISTICS_COLUMN_START = 9;
+    private static final int STATISTICS_ROW_START = 0;
+    private final StatisticService statisticService = new StatisticService();
+
+    @Override
+    public boolean writeTournamentInformation(Tournament tournament) {
+        CellStyle borderCellStyle = ExcelUtil.createCellStyle(workbook, IndexedColors.GOLD.getIndex());
+        CellStyle fillingTableCellStyle = ExcelUtil.createCellStyle(workbook, IndexedColors.LEMON_CHIFFON.getIndex());
+
+        List<Statistics> statistics = tournament.getTeamList().stream()
```

Comment:
jak juz po kropce robisz nowa linie to rob to konsekwentnie, czyli najlepiej zaczac od stream w nowej linijce i kontynuowac